### PR TITLE
Convert  v1.9 to v2.0

### PR DESCRIPTION
**Pull request by MikeAlhayek**
Part of #15892

## Summary by CodeRabbit

- **New Features**
  - Upgraded to version 2.0.0 with significant enhancements.
- **Documentation**
  - Added upgrade instructions and details on breaking changes in the new release.
- **Breaking Changes**
  - Dropped support for `Newtonsoft.Json`.

### CODE REVIEW

File: src/docs/releases/2.0.0.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
-# Orchard Core 1.9.0
+# Orchard Core 2.0.0
 
 Release date: Not yet released
 
+## Upgrade Instructions
+
+Before upgrading to OrchardCore, you should first upgrade to 1.8, address any warning, then upgrade to 2.0. Be sure to review the list of breaking changes documented below to ensure seamless transition.
```
Comment (Piedone) [inline src/docs/releases/2.0.0.md:7]:
```suggestion
Before upgrading to Orchard Core 2.0, you should first upgrade to 1.8, address any warninga, and only then upgrade to 2.0. Be sure to review the list of breaking changes documented below to ensure a seamless transition.
```

**Comment by Piedone** [comment]
Change this: https://github.com/OrchardCMS/OrchardCore/blob/main/mkdocs.yml#L55

**Comment by MikeAlhayek** [comment]
passed! thank you

File: src/docs/releases/2.0.0.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
-# Orchard Core 1.9.0
+# Orchard Core 2.0.0
```
Comment (Piedone) [inline src/docs/releases/2.0.0.md:1]: `EmailMigrations` still references 1.9.

**Review by Piedone — APPROVED** [review]
Looks good after the nit and resolving merge conflicts.

**Pull request merged**